3.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function (string, index) {
+  const stringArray = string.split('')
```

Comment:
Consider chaining these methods for a more concise solution, and to avoid the introduction of temporary variables.